lib/fs_utils/write.js

```diff
@@ -118,7 +119,26 @@ const checkWritten = (fileList, files, startTime) => {
   });
 };
 
-const write = (fileList, config, joinConfig, optimizers, startTime) => {
+const writeStatics = (fileList, config, startTime) => {
+  const allFiles = Array.from(fileList.files.values());
+  const statics = allFiles.filter(f => f.type === 'static' && f.compilationTime >= startTime);
+  const validStatics = statics.map(stat => {
+    const relPath = sysPath.relative(config.files.statics.rootPath, stat.path);
+    if (relPath.slice(0, 2) === '..') {
+      logger.warn(`Resolved ${stat.path} as ${relPath}, skipping. Relative path of a static file should never start with '..'. Possible fix: change 'config.files.statics.rootPath'.`);
```

Comment:
Yeah. If `files.statics.rootPath` is `app` (default) and there is a matching file in say `test`, its relative path will be `../test/filename` — which is at all undesired, since by joining it with `public`, that we'd obtain would be... `test/filename`, not something in `public`